tests/test_importer_utils.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+import unittest
+import WLE_import.importer_utils as utils
+
+
+class TestDictionaryMethods(unittest.TestCase):
+    """Tests for dictionary manipulations."""
+
+    def test_remove_dic_from_list_by_value(self):
+        in_dicts = [{"foo": "mjau", "x": 12}, {"foo": "bar", "x": 44}]
+        out_dicts = [{"foo": "mjau", "x": 12}]
+        self.assertEqual(utils.remove_dic_from_list_by_value(
```

Comment:
Changed it though I cannot say anything about whether it's easier to read! There's an actual science that measures those things.